package.json

```diff
@@ -35,9 +35,10 @@
     "async": "1.5.2",
```

Comment:
Is there any real change in the file? Or maybe the reordering is a side-effect of adding boolean package in one commit, removing in a next one?

(No need to change to make the file exactly at is was, this is just to confirm)

Reply:
Nothing has changed actually, as you mentioned, it's just the result of adding and removing boolean package.

Reply:
Ok. NTC